worlds/peaks_of_yore/options.py

```diff
@@ -0,0 +1,72 @@
+from Options import Toggle, DefaultOnToggle, DeathLink, Choice, PerGameCommonOptions
+from dataclasses import dataclass
+
+
+class Goal(Choice):
+    """
+    Decide what needs to happen to be counted as a win
+
+    **all peaks**: Finish all peaks in selected books to win/complete your run
+    **all artefacts**: Collect all artefacts on selected peaks
+    **all artefacts all peaks**: complete both of the above
+    **all**: all items including bird seeds and ropes
+    """
+    display_name = "Goal"
+    option_all_peaks = 0
+    option_all_artefacts = 1
+    option_all_artefacts_all_peaks = 2
+    option_all = 3
+    default = 0
+
+
+class StartingBook(Choice):
+    """Choose what book to start with. If the book is not enabled, the easiest enabled book will be chosen"""
+    display_name = "Starting Book"
+    option_fundamentals = 0
+    option_intermediate = 1
+    option_advanced = 2
+    option_expert = 3
+    default = 0
+
+
+class StartWithBarometer(DefaultOnToggle):
+    """Choose to start with the barometer, to locate items quicker"""
+    display_name = "Start With Barometer"
+
+
+class EnableFundamental(DefaultOnToggle):
+    """Enables Fundamentals book, items and collectibles"""
+    display_name = "Fundamental Peaks"
+
+
+class EnableIntermediate(DefaultOnToggle):
+    """Enables Intermediate book, items and collectibles"""
+    display_name = "Intermediate Peaks"
+
+
+class EnableAdvanced(DefaultOnToggle):
+    """Enables Advanced book, items and collectibles"""
+    display_name = "Advanced Peaks"
+
+
+class EnableExpert(DefaultOnToggle):
+    """Enables Expert book, items and collectibles"""
+    display_name = "Expert Peaks"
+
+
+class DisableSolemnTempest(DefaultOnToggle):
+    """Removes Solemn Tempest from the locations pool, has no effect if \"Enable Expert Peaks\" is disabled"""
+    display_name = "Disable Solemn Tempest"
+
+
+@dataclass
+class PeaksOfYoreOptions(PerGameCommonOptions):
```

Comment:
Would recommend adding in `start_inventory_from_pool` now that you have `get_filler_item_name` defined.
To do so, add `StartInventoryPool` to your imports from Options,
then in the dataclass, put in `start_inventory_from_pool: StartInventoryPool`
And that's literally all you need to do.

The option is like start inventory, but it pulls the items out of the item pool instead of making new copies of them, and replaces them with a randomly generated filler item from your world.